### ISSUE TITLE
Statemint Deployment Process Drill

### ISSUE THREAD

**Issue by apopiak** [paritytech/cumulus#653]
We want to do a trial run of the release process by deploying Shell on Rococo and upgrading it to Statemint/e.

#### Todos
- [x] remove stale/defunct Shell/Statemint from Rococo
- [x] deploy Shell to Rococo
- [x] set genesis storage on Shell
- [x] upgrade Shell to Statemint/e

**Comment by NachoPal** [comment]
I think we can close this @apopiak ?

**Comment by apopiak** [comment]
well we do want to test the upgrade I think, maybe reword this issue?
